packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -107,6 +111,11 @@ interface BaseConfirmationModalProps {
    */
   readonly cancelLabel?: string;
 
+  /**
+   * Action Type (Work or destructive) for confirm button.
+   */
+  readonly actionType?: ActionType;
```

Comment:
I think that the name `actionType` might be a bit to specific, i think something more semantic like just `type` would be more appropriate. 

Reason being is that if we were to sometime in the future change the color of the text to red, we could do that by saying `type="destructive"`. Whereas `actionType="destructive"` would only describe the button.

Reply:
Another note here is that you have a type issue. You are saying that `ActionType` is a type of `type` and then `actionType` is a type of `ActionType`. To use this prop we would need to do

```
actionType={{type: 'destructive'}}
```

I think it makes more sense to just have `readonly type?: "work" | "descructive"`

Reply:
👍🏻